Makefile

```diff
@@ -2,12 +2,13 @@
 
 # programs
 MAKE=make
-AR=ar
+AR?=ar
```

Comment:
Yes, in case your environment has everything set, you just need to run ```make```. Especially when doing crosscompilation it is important not to call e.g. ```ar``` but the target platform specific version.